po/reuse-docs.pot

```diff
@@ -0,0 +1,1858 @@
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text
+msgid "If the license allows the recipient to [use, study, share and improve](https://fsfe.org/freesoftware/basics/4freedoms.html) the software, then that software is [Free Software](https://fsfe.org/about/basics/freesoftware)."
+msgstr ""
+
+#. type: Title ##
+#: faq.md
+#, markdown-text, no-wrap
+msgid "Which license should I choose? {#which-license}"
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text
+msgid "Always choose a Free Software license, i.e., a license that gives the recipient the freedom to use, study, share, and improve the software. Aside from that, the license you choose is up to you.  If you are contributing to an existing project, you should release your changes under the same license as the project. Otherwise, the [Free Software Foundation](https://www.gnu.org/licenses/license-recommendations.html), [choosealicense.com](https://choosealicense.com/), and [joinup.eu](https://joinup.ec.europa.eu/collection/eupl/joinup-licensing-assistant-jla)  have some good recommendations. Note that these resources each emphasise a different value, and come with their own biases."
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text
+msgid "Above all, if you value freedom, you should choose a license that makes your software [Free Software](https://fsfe.org/about/basics/freesoftware.en.html)."
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text, no-wrap
+msgid "<!-- ## Do I really need to include the license and copyright headers in all files? Why should I care? {#why-care}\n"
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text, no-wrap
+msgid "TODO -->\n"
+msgstr ""
+
+#. type: Title ##
+#: faq.md
+#, markdown-text, no-wrap
+msgid "What is SPDX? {#what-is-spdx}"
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text
+msgid "SPDX stands for [Software Package Data Exchange](https://spdx.org/). It is a project by the [Linux Foundation](https://www.linuxfoundation.org/) and the rock upon which REUSE is built. SPDX defines a standardized way to share copyright and licensing information between projects and people. Most importantly for REUSE, SPDX maintains the [SPDX License List](https://spdx.org/licenses/), which defines standardized identifiers for a lot of licenses."
+msgstr ""
+
+#. type: Title #
+#: faq.md
+#, markdown-text, no-wrap
+msgid "The tool {#tool}"
+msgstr ""
+
+#. type: Title ##
+#: faq.md
+#, markdown-text, no-wrap
+msgid "How do I install and use the REUSE tool? {#install-tool}"
+msgstr ""
+
+#. type: Plain text
+#: faq.md
+#, markdown-text
+msgid "The REUSE tool is a handy companion that can verify your compliance with REUSE and automate a lot of adjacent tasks. You can find the full documentation for the tool at <https://reuse.readthedocs.io/>. But the short of it is:"
+msgstr ""
+
+#. type: Fenced code block
+#: faq.md
+#, no-wrap
+msgid ""
+"$ pip3 install --user fsfe-reuse \n"
```

Comment:
Does code belong into a translation?

Reply:
code often contains translatable bits, like comments or example names like `/path/to/new/file`.  This example does not, but again, technical translators need to be aware of syntax, or they shouldn't be doing technical translation.